packages/nodes-base/credentials/SendInBlueApi.credentials.ts

```diff
@@ -0,0 +1,34 @@
+import {
+	IAuthenticateGeneric,
+	ICredentialTestRequest,
+	ICredentialType,
+	INodeProperties,
+} from 'n8n-workflow';
+
+export class SendInBlueApi implements ICredentialType {
+	name = 'sendinblueApi';
```

Comment:
The name is wrong. Would have to be "sendInBlueApi". Maybe also check with @RicardoE105 I thought we do not use the "Api" at the end anymore.

Reply:
@janober 
It looks to be in the auto-fixable [lint rules](https://github.com/ivov/eslint-plugin-n8n-nodes-base/blob/master/docs/rules/cred-class-field-name-unsuffixed.md)

Reply:
https://github.com/ivov/eslint-plugin-n8n-nodes-base/issues/94